ocaml/xapi/xapi_cluster_host.ml

```diff
@@ -136,6 +136,10 @@ let enable ~__context ~self =
       let network = Db.Cluster.get_network ~__context ~self:cluster in
       let pif = pif_of_host ~__context network host in
       assert_pif_prerequisites pif;
+
+      let pool = Helpers.get_pool ~__context in
+      if Db.Pool.get_ha_enabled ~__context ~self:pool
```

Comment:
The code is fine and indentation is according to `ocp-indent`. However, I feel the code would be clearer with the conditional part indented, especially since there is no else branch.

```
if Db.Pool.get_ha_enabled ~__context ~self:pool then
  Db.Pool.set_ha_cluster_stack ~__context ~self:pool ~value:"corosync";
```
```